RELEASE_NOTES.md

```diff
@@ -49,3 +49,25 @@ PolicyFile users on Chef-13 should be using Chef Server 12.3 or higher.
 
 The remediation is removing the self-dependency `depends` line in the metadata.
 
+### Magic removed from logger and formatter settings
+
+There is no more magic involved in the setting of the logger and the formatter, the default is now the formatter.
```

Comment:
I feel like "magic" isn't a great word to be putting in front of our users. Maybe:

"Make the logging system more obvious" or "Clarify where log messages will be outputted to"?